### ISSUE TITLE
High CPU Useage without any backup or restore

### ISSUE THREAD

**Issue by saviodsouza** [duplicati/duplicati#2570]
I have:
- [Y ] searched open and closed issues for duplicates

----------------------------------------

### Version info

**Duplicati Version:** 2.0.1.67
**Operating System:*Windows Server 2008 R2*
**Backend:**

### Bug description
Duplicati Server.exe Consumes 25% CPU of 4 cores when ide.

### Steps to reproduce

Install duplicati as a service.
I am not sure if this effecting privious versions of duplicati.

**Expected result:** No CPU load

**Comment by kenkendk** [comment]
I have seen similar reports before, and it seems to be something related to timezone settings and having scheduled backups.

**Comment by ts678** [comment]
I looked through the GitHub history of scheduler changes, and I wonder if the above comment is thinking about this issue:

[WebUI schedule can make Duplicati server busy #1635](https://github.com/duplicati/duplicati/issues/1635)

The problem with this line of thinking is that the fix went in [2.0.1.18_canary_2016-08-10](https://github.com/duplicati/duplicati/releases/tag/v2.0.1.18-2.0.1.18_canary_2016-08-10) and I think yours is a later version.

> Fixed an issue with the scheduler that cause high CPU usage for invalid schedules

I searched open and closed issues for newer ones with `cpu` in the title, and ignored ones with specific actions not said here.

Came up empty, and I saw that 2017 seemed to be the peak year for this sort of complaint. Is it still of interest and around?

I don't think there's enough information in this issue so far to pursue this further. Issue will close automatically if it stays idle.
